ls1intum/Artemis
# `Assessment`: Fix star ratings functionality

### Checklist
#### General


#### Client

### Motivation and Context


After removing some libraries, the star rating component didn't work anymore.

### Description

Added getters for all values, and it works now.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Exercise with Feedback

1. Log in to Artemis
2. Navigate to the exercise submission
3. Check that the rating component is displayed correctly
4. Give a rating
5. Check that the rating component works properly

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots


<img width="860" alt="Bildschirmfoto 2022-01-05 um 11 44 47" src="https://user-images.githubusercontent.com/51987021/148228236-8766bc4e-4528-40b7-92bc-6b765960f552.png">
